webanno-io-csv/pom.xml

```diff
@@ -61,6 +63,48 @@
     	<groupId>org.springframework</groupId>
     	<artifactId>spring-context</artifactId>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>de.tudarmstadt.ukp.dkpro.core</groupId>
+      <artifactId>de.tudarmstadt.ukp.dkpro.core.testing-asl</artifactId>
+      <scope>test</scope>
+      <!--
+      FIXME
+        manual adding of version necessary in this case since the managed version (1.11.0)
+        is not released yet
+      -->
+      <version>1.10.0</version>
```

Comment:
Actually, it is released, but the groupIds have changed. See the release notes: https://github.com/dkpro/dkpro-core/releases/tag/dkpro-core-1.11.0